AppCenterCrashes/AppCenterCrashes/MSCrashes.mm

```diff
@@ -277,6 +279,22 @@ - (instancetype)init {
      * Setting this flag will let application crash on uncaught exceptions.
      */
     [[NSUserDefaults standardUserDefaults] registerDefaults:@{ @"NSApplicationCrashOnExceptions" : @YES }];
+
+    /*
+     * Exceptions on the main thread of a Cocoa application do not typically rise to the level of
+     * the uncaught exception handler because the global application object catches all such exceptions.
+     * See: https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/Exceptions/Concepts/UncaughtExceptions.html
+     */
+    if (![MSAppCenter isDebuggerAttached]) {
+      SEL selector = @selector(reportException:);
```

Comment:
Some people just don't like swizzling so we usually provide a way to opt-out.

Reply:
HockeyApp has the way without swizzling, but "one line to integrate the SDK" is better approach I guess.

Reply:
We already have a Info.plist to disable swizzling, the default is still swizzling.
We should make sure we read that existing flag and document.

Reply:
that it isn't swizzling App**Delegate** but `NSApplication`, so it doesn't look like `AppCenterAppDelegateForwarderEnabled` is good for the case